app/src/lumen/java/com/cliqz/browser/purchases/PurchasesManager.kt

```diff
@@ -93,4 +93,15 @@ class PurchasesManager(
         serverData != null && serverData!!.isInTrial
     }
 
+    fun getSubscriptionTypeForTelemetry() = when {
```

Comment:
```
    fun getSubscriptionTypeForTelemetry() = when {
        purchase.isASubscriber && purchase.isDashboardEnabled && purchase.isVpnEnabled -> "basic_vpn"
        purchase.isASubscriber && purchase.isDashboardEnabled -> "basic"
        purchase.isASubscriber -> "vpn"
        serverData?.isInTrial ?: false -> "trial"
        else -> "free"
    }
```

Reply:
Probably, it will be better to add synthetic properties like `isABasicUser`, `isAVpnUser`, `isAFullUser`